Copy to SD Card root directory to update/config_rrf.ini

```diff
@@ -80,7 +80,7 @@ status_screen:1
 baudrate:4
 
 #### Enable Multi-Serials
-# Multi-Serials connected to ESP3D, OctoPrint, Other TFT, etc...
```

Comment:
"etc..." is present also on the other two config.ini file. It should be better to apply the same change also on them (so "etc.")

Reply:
ok